services/autorust/codegen/examples/gen_workspace.rs

```diff
@@ -0,0 +1,25 @@
+// cargo run --example gen_workspace
+
+use autorust_codegen::{
+    crates::{list_crate_names, list_dirs},
+    jinja::{CargoToml, CheckAllServicesYml},
+};
+pub type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+fn main() -> Result<()> {
+    let dirs = list_dirs()?;
+    let dirs: Vec<String> = dirs.iter().map(|dir| dir.as_str().replace("\\", "/").replace("../", "")).collect();
+
+    let yml = CargoToml { dirs };
+    yml.create("../Cargo.toml")?;
```

Comment:
Is there a benefit of writing out all of the crate names, rather than using wildcards?  I imagine this will just cause churn in git without providing value.